README.rst

```diff
@@ -202,4 +202,52 @@ again with the command::
 
     bors retry
 
-For more details, see `the bors reference docs <https://bors.tech/documentation/>`_
\ No newline at end of file
+For more details, see `the bors reference docs <https://bors.tech/documentation/>`_
+
+Releasing
+---------
+
+We follow the usual ``zest.releaser`` approach for releases.
+
+First:
+
+- Make sure the CHANGELOG is up-to-date and includes details about all the components included in the release
+
+.. code-block:: bash
+
+  $ git checkout -b prepare-X.Y.Z
+  $ prerelease
+
+Then:
+
+- Open a PR
+- When the PR is approved, merge it using bors
+
+Then:
+
+.. code-block:: bash
+
+   $ git checkout master
+   $ git pull
+   $ release
+   $ postrelease
```

Comment:
Add a note telling this will be deployed to STAGE at Mozilla?

Reply:
This is only true for remote-settings I think.

Reply:
But it's easy to add a note that clarifies that if you want!

Reply:
That's fine for me